charts/backingservices/README.md

```diff
@@ -31,18 +31,19 @@ If an externally managed elasticsearch cluster is being used, make sure the serv
 You may enable the component of [Elasticsearch](https://github.com/helm/charts/tree/master/stable/elasticsearch/values.yaml) in the backingservices by configuring the 'srs.srsStorage' section in values.yaml file to deploy ElasticSearch cluster automatically. For more configuration options available for each of the components, see their Helm Charts.
 
 Note: Pega does **not** actively update the elasticsearch dependency in `requirements.yaml`. To leverage SRS, you should:
-* Update the elasticsearch `version` value, or
+
+* Update the elasticsearch `version` value (the version value in `requirements.yaml` should be same as imageTag in [values.yaml](./values.yaml)), or
 * Disable deploying elasticsearch by setting `srs.srsStorage.provisionInternalESCluster` to false.
```

Comment:
* To use an externally-provided Elasticsearch service from the SRS cluster, update the `srs.srsStorage.provisionInternalESCluster` parameter in the [values.yaml](./values.yaml) to `false` and then provide connection details as documented below.

Reply:
sorry but I missed these bullets in previous iterations of this readme ;-/